ts/src/pro/bitmart.ts

```diff
@@ -867,6 +840,10 @@ export default class bitmart extends bitmartRest {
         if (data === undefined) {
             return;
         }
+        const isSwap = ('group' in message);
+        if (isSwap) {
+            data.reverse (); // fix chronological order
```

Comment:
again I'm not a big fan of this reverse, when we can easily iterate backwards without iterating twice over the array

Reply:
ah yes, this update was before you gave me correct idea about reversing. now fixed ✌️